theme/templates/pages/group.html

```diff
@@ -96,6 +96,7 @@ <h4 class="flag-joined"><span class="glyphicon glyphicon-send"></span> Request
                                     {% csrf_token %}
                                     <button type="submit" class="btn btn-primary" role="button">Ask to join
                                     </button>
+                                    {% comment %}TODO: do we need to add explanation abilities on this join?{% endcomment %}
```

Comment:
There is a TODO in here.  Is this for the group landing page?  If so, we will need the ability to add explanations here too.